vmdb/app/models/metric/capture.rb

```diff
@@ -110,14 +110,16 @@ def self.perf_capture_timer(zone = nil)
     targets.each do |target|
       interval_name = self.perf_target_to_interval_name(target)
 
-      parent = target.perf_rollup_parent(interval_name)
       options = {}
-      if parent && tasks_by_rollup_parent.has_key?("#{parent.class}:#{parent.id}")
-        pkey = "#{parent.class}:#{parent.id}"
-        tkey = "#{target.class}:#{target.id}"
-        if targets_by_rollup_parent[pkey].include?(tkey)
-          options[:task_id] = tasks_by_rollup_parent[pkey].id
-          options[:force]   = true # Force collection since we've already verified that capture should be done now
+      (target.perf_rollup_parents(interval_name) || []).each do |parent|
+        if tasks_by_rollup_parent.key?("#{parent.class}:#{parent.id}")
+          pkey = "#{parent.class}:#{parent.id}"
+          tkey = "#{target.class}:#{target.id}"
+          if targets_by_rollup_parent[pkey].include?(tkey)
+            # FIXME: check that this is still correct
```

Comment:
@Fryguy any advice here? In the case of multiple parents we can set the following options only with one value. Would it be still correct?